calicoctl/commands/node/diags.go

```diff
@@ -235,3 +243,39 @@ func writeDiags(cmds diagCmd, dir string) {
 		log.Errorf("Error writing diags to file: %s\n", err)
 	}
 }
+
+func walkIPTables(file string, command string)([]byte) {
```

Comment:
Looks like you want to run `goimports` across this file to fix up some minor formatting / spacing issues.